src/triggers/TDTM_Account.trigger

```diff
@@ -30,8 +30,6 @@
 trigger TDTM_Account on Account (after delete, after insert, after undelete, 
 after update, before delete, before insert, before update) {
 
-    TDTM_TriggerHandler handler = new TDTM_TriggerHandler();  
-    handler.run(Trigger.isBefore, Trigger.isAfter, Trigger.isInsert, Trigger.isUpdate, Trigger.isDelete, 
-        Trigger.isUnDelete, Trigger.new, Trigger.old, Schema.Sobjecttype.Account, 
-        new TDTM_ObjectDataGateway());
+    TDTM_Global_API.run(Trigger.isBefore, Trigger.isAfter, Trigger.isInsert, Trigger.isUpdate, Trigger.isDelete, 
```

Comment:
We might as well "drink our own champagne", and go through the TDTM_Global_API class. It will also make more sense for those that look at the codebase.